.eslintrc.js

```diff
@@ -118,16 +135,8 @@ const tslintRules = {
     true,
     "double"
   ],
-  "react-no-dangerous-html": [
-    true,
-    [
-      {
-        "file": "./src/app/containers/Html.tsx",
-        "method": "render",
-        "comment": "string comes from trusted source - our initial state"
-      }
-    ]
-  ],
+   // todo: Specifying excluded files as an option used to work until updating eslint and it's parsers.
+  "react-no-dangerous-html": true,
```

Comment:
As commented, can't get to work after update eslint and parsers. And not surprisingly the documented version is still [not working](https://github.com/microsoft/tslint-microsoft-contrib/issues/880) and I don't expect it to be fixed. [eslint plugin react no danger rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-danger.md) also does not have the option to specify excluded files/locations.